TestResultSummaryService/parsers/Parser.js

```diff
@@ -141,7 +141,7 @@ class Parser {
         let skipped = 0;
         let disabled = 0;
         const summaryRegex =
-            /\S*\s*?TOTAL:\s*([0-9]*)\s*EXECUTED:\s*([0-9]*)\s*PASSED:\s*([0-9]*)\s*FAILED:\s*([0-9]*)\s*DISABLED:\s*([0-9]*)\s*SKIPPED:\s*([0-9]*)\s*/;
```

Comment:
Might be worth adding a comment showing some samples of how this regex is applied

Reply:
Updated, thank you.